Gemfile

```diff
@@ -53,10 +53,12 @@ group :test do
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
+gem 'tzinfo-data'
 
 gem 'net-imap'
 gem 'net-pop'
 gem 'net-smtp'
 
 gem 'carrierwave'
+gem 'kaminari'
+gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```

Comment:
kaminari以外の部分はなぜ必要になったのでしょうか？

Reply:
rails sの実行後このように警告が出たからですが、コミットに含めるべきではありませんでした。
修正しておきます
```
> C:/Ruby32-x64/lib/ruby/gems/3.2.0/gems/tzinfo-2.0.4/lib/tzinfo/data_source.rb:159:in `rescue in create_default_data_source': tzinfo-data is not present. Please add gem 'tzinfo-data' to your Gemfile and run bundle install 
> (TZInfo::DataSourceNotFound)

>   Please add the following to your Gemfile to avoid polling for changes:
>     gem 'wdm', '>= 0.1.0' if Gem.win_platform?
>   Please add the following to your Gemfile to avoid polling for changes:
>     gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```

Reply:
@anada-being 
Windowsで開発する場合、フィヨルドブートキャンプではWSL2を使うことを推奨しています。
https://bootcamp.fjord.jp/articles/9

そのままWindowsで実行する場合、今回のような固有の問題にハマりやすく、プラクティスの本質的な部分以外のところで時間をとられるからです。
WSL2環境の導入を検討してもらってもよいでしょうか。

Reply:
wsl2で試してエラーが出たので、Windowsで試し、どちらもエラーが出たため、wsl2の環境設定による影響が余分に起こるかもしれないと思い、windowsで取り組んでいました。
本日再度wsl2で行うとREAD.meの手順で正常に動作したため、以降wsl2で作業させていただきます。